src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDurationMappingTest.java

```diff
@@ -83,54 +100,36 @@ private void adjournmentFlagEnabled(boolean value) {
     @DisplayName("When a valid adjournCaseDuration and adjournCaseDurationUnits is given "
         + "getHearingDuration returns the correct duration Parameterized Tests")
     @ParameterizedTest
-    @CsvSource(value = {
-        "120,minutes,120",
-        "70,minutes,70",
-        "1,sessions,165",
-        "2,sessions,330",
-        "3,sessions,495",
-    }, nullValues = {"null"})
-    void getHearingDuration(String adjournCaseDuration, String adjournCaseDurationUnits, int expected) {
+    @MethodSource
+    void getHearingDuration(Integer adjournCaseDuration, AdjournCaseNextHearingDurationUnits adjournCaseDurationUnits, int expected) {
         adjournmentFlagEnabled(true);
 
-        SscsCaseData caseData = adjourningCaseBuilder(adjournCaseDuration, adjournCaseDurationUnits);
+        SscsCaseData caseData = adjourningCaseBuilder(
+            adjournCaseDuration,
+            adjournCaseDurationUnits
+        );
 
-        int result = HearingsDurationMapping.getHearingDuration(caseData, referenceDataServiceHolder);
+        Integer result = HearingsDurationMapping.getHearingDuration(caseData, referenceDataServiceHolder);
 
         assertThat(result).isEqualTo(expected);
     }
 
-    @DisplayName("When an invalid adjournCaseDuration or adjournCaseDurationUnits is given "
-        + "getHearingDuration returns the default duration Parameterized Tests")
```

Comment:
I don't think this test is required any more as it is not possible to choose invalid units